src/layouts/Page.tsx

```diff
@@ -194,13 +221,14 @@ export const Page = (props: Props) => {
           }}
           balance={balance}
           currency={currency} />
-        {!!props.headerComponent &&
-          props.headerComponent
-        }
-        {walletState !== 'WRONG_NETWORK' ? props.children : <CircleLoader />}
-        {!!props.footerComponent &&
-          props.footerComponent
+        {props.headerComponent}
+        {
+          walletState !== 'WRONG_NETWORK'
```

Comment:
Would be nice show something you can show when wallet has state "WRONG_NETWORK" instead of showing loader until the user doesn't switch a network.